### ISSUE TITLE
SDK: implement requirements for Baseline integration

### ISSUE THREAD

**Issue by 0xJem** [Axis-Fi/launchpad#60]

**Comment by edmulraney** [comment]
@0xJem what baseline functionality did you want to add to the sdk? is it creating an auction with baseline callbacks preconfigured?

**Comment by 0xJem** [comment]
Yes, but not really feasible until permissionless Baseline tokens are possible
